src/mainviews/job-detail.tsx

```diff
@@ -1,13 +1,398 @@
+                      label={trans.__('Tag')}
+                      defaultValue={tag}
+                      InputProps={{
+                        readOnly: true
+                      }}
+                    />
+                  ))}
+                <FormLabel component="legend">
+                  {trans.__('Email Notifications')}
+                </FormLabel>
+                <TextFieldStyled
+                  label={trans.__('Job Definition Id')}
+                  defaultValue={job?.timeout_seconds?.toString() ?? ''}
+                  InputProps={{
+                    readOnly: true
+                  }}
+                />
+                <TextFieldStyled
+                  label={trans.__('max_retries')}
+                  defaultValue={job?.max_retries?.toString() ?? ''}
+                  InputProps={{
+                    readOnly: true
+                  }}
+                />
+                <TextFieldStyled
+                  label={trans.__('min_retry_interval_millis')}
+                  defaultValue={
+                    job?.min_retry_interval_millis?.toString() ?? ''
+                  }
+                  InputProps={{
+                    readOnly: true
+                  }}
+                />
+                <TextFieldStyled
+                  label={trans.__('retry_on_timeout')}
+                  defaultValue={job?.retry_on_timeout?.toString() ?? ''}
+                  InputProps={{
+                    readOnly: true
+                  }}
+                />
+                <TextFieldStyled
+                  label={trans.__('start_time')}
+                  defaultValue={job?.start_time?.toString() ?? ''}
+                  InputProps={{
+                    readOnly: true
+                  }}
+                />
+                <TextFieldStyled
+                  label={trans.__('output_filename_template')}
+                  defaultValue={job?.output_filename_template ?? ''}
+                  InputProps={{
+                    readOnly: true
+                  }}
+                />
+
+                <TextFieldStyled
+                  label={trans.__('url')}
+                  defaultValue={job?.url ?? ''}
+                  InputProps={{
+                    readOnly: true
+                  }}
+                />
+                <TextFieldStyled
+                  label={trans.__('status_message')}
+                  defaultValue={job?.status_message ?? ''}
+                  InputProps={{
+                    readOnly: true
+                  }}
+                />
+                <TextFieldStyled
+                  label={trans.__('start_time')}
+                  defaultValue={job?.start_time?.toString() ?? ''}
+                  InputProps={{
+                    readOnly: true
+                  }}
+                />
+                <TextFieldStyled
+                  label={trans.__('end_time')}
+                  defaultValue={job?.end_time?.toString() ?? ''}
```

Comment:
This should be converted from epoch seconds to a human-readable format.

Reply:
Done, adapted your solution